README.yaml

```
@@ -81,7 +81,7 @@ description: |-
 usage: |-
   ```hcl
   module "alb_alarms" {
-    source                  = "git::https://github.com/cloudposse/terraform-aws-alb-target-group-cloudwatch-sns-alarms.git?ref=tags/0.1.0"
+    source                  = "git::https://github.com/cloudposse/terraform-aws-alb-target-group-cloudwatch-sns-alarms.git?ref=tags/0.5.1"
```

```suggestion
    source                  = "git::https://github.com/cloudposse/terraform-aws-alb-target-group-cloudwatch-sns-alarms.git?ref=master"
```

@Eascen, we usually use `master` in examples because it's difficult to sync the docs with the latest version of the modules. On the other hands, we recommend pinning to a release and not using the master branch in TF code for stability

---

@aknysh The reason I set a specific version is to ensure that the example works properly. The issue I ran into was that the example used 0.1.0 pinned in the version in the example usage, but the example parameters did not function. If updated to master, it could also assume that documentation and code might be out of sync, I'd be happy to make the change though.